src/main/kotlin/no/nb/mlt/wls/product/payloads/ApiProductPayload.kt

```diff
@@ -7,7 +7,6 @@ import no.nb.mlt.wls.core.data.Owner
 import no.nb.mlt.wls.core.data.Packaging
 import no.nb.mlt.wls.product.model.Product
 
-// TODO - Enforce style with validation. Should valid fields be returned in an error message, or be declared in the description?
 @Schema(
     description = "Payload for registering a product in Hermes WLS, and appropriate storage system for the product.",
     example = """
```

Comment:
Just a small thing, we should remove location and quantity from the example, I goofed up when I wrote it